### ISSUE TITLE
Application doesn't launch on Linux

### ISSUE THREAD

**Issue by sweiland** [hoedlale16/fhj.swengb2017.assignments#1]
On startup, the following exception is thrown.
`Exception in Application init method
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:389)
	at com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:328)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:767)
Caused by: java.lang.RuntimeException: Exception in Application init method
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:912)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$1(LauncherImpl.java:182)
	at java.lang.Thread.run(Thread.java:748)
Caused by: MediaException: UNKNOWN : com.sun.media.jfxmedia.MediaException: Could not create player! : com.sun.media.jfxmedia.MediaException: Could not create player!
	at javafx.scene.media.MediaException.exceptionToMediaException(MediaException.java:146)
	at javafx.scene.media.MediaPlayer.init(MediaPlayer.java:511)
	at javafx.scene.media.MediaPlayer.<init>(MediaPlayer.java:414)
	at at.fhj.swengb.apps.battleship.model.BattleShipJukeBox.<init>(BattleShipJukeBox.scala:9)
	at at.fhj.swengb.apps.battleship.jfx.BattleShipFxApp.init(BattleShipFxApp.scala:130)
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:841)
	... 2 more
Caused by: com.sun.media.jfxmedia.MediaException: Could not create player!
	at com.sun.media.jfxmediaimpl.NativeMediaManager.getPlayer(NativeMediaManager.java:274)
	at com.sun.media.jfxmedia.MediaManager.getPlayer(MediaManager.java:118)
	at javafx.scene.media.MediaPlayer.init(MediaPlayer.java:467)
	... 6 more
Exception running application at.fhj.swengb.apps.battleship.jfx.BattleShipFxApp`

**Comment by hoedlale16** [comment]
Not a code-problem: Environment problem:
https://stackoverflow.com/questions/24090356/javafx-mediaplayer-could-not-create-player-error-in-ubuntu-14-04

**Comment by sweiland** [comment]
tl;dr: ffmpeg-compat library must be installed
